backend/lib/models/Post.js

```diff
@@ -64,8 +64,8 @@ const postSchema = new Schema(
 );
 
 // -- Indexes
-/* eslint-disable */
 // Indexes for filtered feed
+/* eslint-disable sort-keys */
```

Comment:
ah ok I thought this was in the lint config now, but this is more clear probably so everyone sees that this part is an exception to the rule